src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -118,433 +131,633 @@ def test_build_flowsheet(self, model):
+            62.01, 1e-4
+        )
+        assert scaled_model.fs.cleaner_sep.inlet.flow_vol[0].value == pytest.approx(
+            62.01, 1e-4
+        )
+        assert scaled_model.fs.leach_sx_mixer.outlet.flow_vol[0].value == pytest.approx(
+            440.422915, 1e-4
+        )
+        assert scaled_model.fs.cleaner_purge.inlet.flow_vol[0].value == pytest.approx(
+            6.201, 1e-4
+        )
+        assert scaled_model.fs.sl_sep1.recovered_liquid_outlet.flow_vol[
+            0
+        ].value == pytest.approx(434.662915, 1e-4)
+        assert scaled_model.fs.sl_sep2.recovered_liquid_outlet.flow_vol[
+            0
+        ].value == pytest.approx(0.063, 1e-4)
+        assert scaled_model.fs.precip_sep.inlet.flow_vol[0].value == pytest.approx(
+            0.063, 1e-4
+        )
+        assert scaled_model.fs.precip_sx_mixer.outlet.flow_vol[
+            0
+        ].value == pytest.approx(5.76, 1e-4)
+        assert scaled_model.fs.precip_purge.inlet.flow_vol[0].value == pytest.approx(
+            0.0063, 1e-4
+        )
+
+    def test_main(self):
+        main()
 
     @pytest.mark.unit
-    def test_costing(self, model):
-        m = add_costing(model)
+    def test_costing(self, scaled_model):
+        m = add_costing(scaled_model)
 
-        assert m.fs.costing.total_plant_cost.value == pytest.approx(15.6094, rel=1e-4)
-        assert m.fs.costing.total_BEC.value == pytest.approx(5.255, rel=1e-4)
-        assert m.fs.costing.total_installation_cost.value == pytest.approx(
-            10.353, rel=1e-4
+        assert value(m.fs.costing.total_plant_cost) == pytest.approx(15.5759, rel=1e-4)
+        assert value(m.fs.costing.total_BEC) == pytest.approx(5.244, rel=1e-4)
+        assert value(m.fs.costing.total_installation_cost) == pytest.approx(
+            10.330, rel=1e-4
         )
-        assert m.fs.costing.other_plant_costs.value == pytest.approx(
+        assert value(m.fs.costing.other_plant_costs) == pytest.approx(
             0.0016309, rel=1e-4
         )
-        assert m.fs.costing.total_fixed_OM_cost.value == pytest.approx(7.2512, rel=1e-4)
-        assert m.fs.costing.total_variable_OM_cost[0].value == pytest.approx(
+        assert value(m.fs.costing.total_fixed_OM_cost) == pytest.approx(
+            7.2484, rel=1e-4
+        )
+        assert value(m.fs.costing.total_variable_OM_cost[0]) == pytest.approx(
             10.7422, rel=1e-4
         )
         assert value(m.fs.costing.land_cost) == pytest.approx(
             6.1234e-5, rel=1e-4
         )  # Expression, not Var
-        assert m.fs.costing.total_sales_revenue.value == pytest.approx(
-            0.35555, rel=1e-4
+        assert value(m.fs.costing.total_sales_revenue) == pytest.approx(
+            0.00018136, rel=1e-4
         )
 
     @pytest.mark.component
-    def test_report(self, model):
-        m = add_costing(model)
+    def test_report(self, scaled_model):
+        m = add_costing(scaled_model)
         display_costing(m)
```

Comment:
This does not look like it is actually asserting anything - i.e. all this tells us is that the code didn't raise an exception.

Reply:
I added this just for code coverage.

Reply:
You need to assert something at least. For `display` type methods you should set up the `display` method to print to a user defined stream so that you can capture it here and assert that it is correct.